bot.py

```diff
@@ -58,12 +59,17 @@
             args = '\\n '.join(message.content[1:].splitlines()).split()
             if args:
                 cls, args = commands.Helpme.find_command(args)
-                output = await cls(client, message).init(*args)
-                if isinstance(output, discord.Embed):
-                    await client.send_message(message.channel, embed=output)
-                elif output is not None:
-                    await client.send_message(message.channel, output)
+                if config['COMMANDS'].getboolean(cls.__name__):
+                    # Command is not disabled
+                    output = await cls(client, message).init(*args)
+                    if isinstance(output, discord.Embed):
+                        await client.send_message(message.channel, embed=output)
+                    elif output is not None:
+                        await client.send_message(message.channel, output)
     except discord.errors.HTTPException as e:
         await client.send_message(message.channel, err)
 
-client.run(os.environ['TOKEN'])
+if config.read('config/config.ini'):
+    client.run(config['KEYS']['DiscordToken'])
+else:
+    print("Can't find config file! Have you added your keys to the default config file and renamed the file to 'config.ini'?")
```

Comment:
Should raise exception instead, bot shouldn't run without a config file (we provide one in the repo). 
You may want to use `read_file()` and `open()` for this instead and extend the `FileNotFound` exception thrown by `open()` to something like `ConfigFileNotFound` with a helpful error message.

Reply:
Exception raised. I don't think it's possible to raise an error with `config.read()` - it just returns False if it was unsuccessful.